.github/scripts/.helm-tests/default/result.yaml

```diff
@@ -158,135 +158,162 @@ spec:
+                    description: ProviderResult stores reference of already collected
+                      provider query associated to its objective template
+                    properties:
+                      errMsg:
+                        description: ErrMsg stores any possible error at retrieval time
+                        type: string
+                      objectiveReference:
+                        description: Objective store reference to corresponding objective
+                          template
+                        properties:
+                          name:
+                            description: Name defines the name of the referenced object
+                            type: string
+                          namespace:
+                            description: Namespace defines the namespace of the referenced
+                              object
+                            type: string
+                        required:
+                          - name
+                        type: object
+                      value:
+                        description: Value is the value the provider returned
+                        type: string
+                    type: object
+                  description: StoredValues contains all analysis values that have already
+                    been retrieved successfully
+                  type: object
+                timeframe:
+                  description: Timeframe describes the time frame which is evaluated
+                    by the Analysis
                   properties:
-                    errMsg:
-                      description: ErrMsg stores any possible error at retrieval time
+                    from:
+                      description: From is the time of start for the query, this field
+                        follows RFC3339 time format
+                      format: date-time
                       type: string
-                    objectiveReference:
-                      description: Objective store reference to corresponding objective
-                        template
-                      properties:
-                        name:
-                          description: Name defines the name of the referenced object
-                          type: string
-                        namespace:
-                          description: Namespace defines the namespace of the referenced
-                            object
-                          type: string
-                      required:
-                      - name
-                      type: object
-                    value:
-                      description: Value is the value the provider returned
+                    recent:
+                      description: Recent describes a recent timeframe using a duration
+                        string. E.g. Setting this to '5m' will cause the Analysis to
+                        be performed for the last five minutes
```

Comment:
```suggestion
                        string. E.g. Setting this to '5m' causes the Analysis to
                        be performed for the last five minutes
```

Reply:
already adapted to the previous suggestion